### DISCUSSION TITLE
When delete cluster, should I delete Cloud-Config Secret?

### DISCUSSION THREAD

**Discussion by youngbo89** [kubernetes-sigs/cluster-api#7131] [Q&A] [answered]
>IMPORTANT: In order to ensure a proper cleanup of your infrastructure you must always delete the cluster object. Deleting the entire cluster template with kubectl delete -f capi-quickstart.yaml might lead to pending resources to be cleaned up manually.

According to documents,  Deleting `cluster` object is enough. Then Cloud-Config also is removed by Cluster-API providers/operators?

Or should be deleted by hands? Thank you in advance.

**Comment by killianmuldoon** [comment, marked as answer]
What provider are you working with? FYI It might be better to open an issue for this instead of a discussion. Issues are tracked and are the main way to report problems and ask questions about Cluster API.
